README.rst

```diff
@@ -145,6 +145,10 @@ The possible filters are:
   ``kinto_signer.events.ReviewRequested``, ``kinto_signer.events.ReviewApproved``,
   ``kinto_signer.events.ReviewRejected``
 
+If a filter value starts with the special character ``"^"``, then the matching will consider the filter value to be a regular expression.
```

Comment:
I don't think you need the quotation marks here -- for me, the backticks indicate a special character.